spec/phlex/component_spec.rb

```diff
@@ -421,4 +421,22 @@ def template
       expect(output).to eq %{<article class="p-5 rounded drop-shadow">Hello World!</article>}
     end
   end
+
+  describe "with a Rails partial" do
+    let(:component) do
+      Class.new Phlex::Component do
+        def template
+          component CardComponent do
+            render "content", locals: { name: "Alexandre" }
+          end
+        end
+      end
+    end
+
+    it "produces the correct markup" do
+      ActionView::Base.with_view_paths(File.expand_path("../../examples/partials", __dir__)) do
```

Comment:
Maybe we should do this globally and not per spec.
I'll have a look if I can find a option that does that.

Reply:
You'll be able to do it with an around each I think.

Reply:
I couldn't find a way to make this work. The following code does not work for instance:

```rb
around(:each) do |example|
  ActionView::Base.with_view_paths(...) do 
    example.run
  end
end
```